packages/app/src/systems/Transaction/components/TxIcon/TxIconLoader.tsx

```diff
@@ -0,0 +1,17 @@
+import { darkColors } from '@fuel-ui/css';
+import type { IContentLoaderProps } from 'react-content-loader';
+import ContentLoader from 'react-content-loader';
+
+export const TxIconLoader = (props: IContentLoaderProps) => (
```

Comment:
I didn't understand why you need a Loader for `TxIcon` component? 🤔 Because since you already have a loader component for the `ActivityItem` I guess you don't need another one here.